data/json/items/gunmod/stock.json

```diff
@@ -123,5 +123,66 @@
     "dispersion_modifier": -1,
     "handling_modifier": 2,
     "min_skills": [ [ "weapon", 2 ] ]
+  },
+  {
+    "id": "butt_hook",
+    "type": "GUNMOD",
+    "name": "butt hook",
+    "description": "An attachment to the butt of the gun that is put under the armpit allowing for greater stability.",
+    "weight": "100 g",
+    "volume": "100 ml",
+    "price": 4750,
+    "price_postapoc": 50,
+    "material": [ "aluminum" ],
+    "symbol": ":",
+    "color": "dark_gray",
+    "location": "stock",
+    "mod_targets": [ "smg", "rifle", "shotgun", "launcher", "crossbow" ],
+    "install_time": "10 m",
```

Comment:
It really requires 10 minutes to attach a hook?

Reply:
It's not a replacement stock, you screw it to the butt of one.
So need to find a position that will fit your arm, then screw it solid.